test/test_universal_qpu.py

```
@@ -0,0 +1,815 @@
+    qpu.client.platforms[0]["meta"] = {"maxQubits": 26}
+
+    assert qpu.input_formats == []
+    qpu.run(FakeGuppyFunction())
+    (program,) = _uploaded_programs(qpu)
+    assert program["serialization_format"] == wire.PROGRAM_HUGR_V1
+
+
+def test_unknown_platform_raises(qpu_mod):
+    qpu = qpu_mod.QPU(platform="nexus:Nope")
+    with pytest.raises(LookupError, match="nexus:Nope"):
+        qpu.input_formats
```

Ruff (0.16.1)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)